alpine.Dockerfile.template

```diff
@@ -15,10 +15,12 @@ COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifi
 
 # run as user for security
 RUN mkdir /unftp
-RUN addgroup -S unftp && adduser -S unftp -G unftp
-COPY --from=builder /usr/src/unftp/target/x86_64-unknown-linux-musl/release/unftp /unftp
-RUN chown -R unftp:unftp /unftp
+RUN addgroup -g 1010 unftp \
```

Comment:
Run as a specific PID 1010

Reply:
should change `USER unftp` to `USER unftp:unftp` to actually run with the unftp gid

Reply:
Won’t it run as such because the user is part of that group ?